x-pack/plugins/alerts/server/task_runner/task_runner.ts

```diff
@@ -153,7 +155,8 @@ export class TaskRunner {
     alert: SanitizedAlert,
     params: AlertExecutorOptions['params'],
     executionHandler: ReturnType<typeof createExecutionHandler>,
-    spaceId: string
+    spaceId: string,
+    event: Event
```

Comment:
We now pass a partially populated event in to these "inner" functions of the execution logic, as we write the event at the "top level" of the execution logic, instead of in this "inner" one.